app/models/kit.rb

```diff
@@ -1,6 +1,7 @@
 class Kit < ApplicationRecord
   has_and_belongs_to_many :kit_items, join_table: "kit_items_kits"
   has_one_attached :image
+  has_many :orders
```

Comment:
Sorry again, If I added more code then needed for the ticket.
The relation "has_many :orders" suppose to be there from the beginning, since we had " belongs_to: kit" in orders when it still was kit_requests. I mentioned it in the discussion with Raquel in HER-17 PR.